app/templates/marketing/terms_of_service.html

```diff
@@ -41,10 +41,15 @@ <h4 class="ui header" id="issue_tracker">Issues or Concerns?</h4>
 
         <h1 class="ui dividing header" id="code_of_conduct">Code of Conduct</h1>
 
+        <p>As a Python community we are beholden to the Python Software Foundation (PSF)
+           <a href="https://www.python.org/psf/codeofconduct/">Code of Conduct</a>.
+        </p>
+        <p>This is extended by our own PySlackers Code of Conduct detailed below:</p>
+
         <h2 class="ui header" id="our_pledge">Our Pledge</h2>
 
         <p>In the interest of fostering an open and welcoming environment, we as
-            contributors and maintainers pledge to making participation in our
+            contributors and maintainers pledge to make participation in our
             project and
             our community a harassment-free experience for everyone, regardless
             of age, body
```

Comment:
suggest changing to "body types", rather than "body size"

also, `ethinicy` and `race` both appear here.  Duplicate?

Reply:
Body sizes now == body types.

Decided race and ethnicity are distinct enough to include together.